publication-rest/src/main/java/no/unit/nva/publication/services/model/UriMap.java

```diff
@@ -0,0 +1,32 @@
+package no.unit.nva.publication.services.model;
+
+import static java.util.Objects.isNull;
+import java.net.URI;
+import java.time.Instant;
+import no.unit.nva.identifiers.SortableIdentifier;
+import nva.commons.core.StringUtils;
+import nva.commons.core.paths.UriWrapper;
+
+public record UriMap(URI shortenedUri, URI longUri, Instant createdDate, Long expiresDate) {
+
+    private static final String SHORTENED_PATH = "file";
+
+    public static UriMap create(URI longVersion, Instant expiresDate, URI domain, String basePath) {
```

Comment:
Why is basepath not part of the mysterious "domain" URI? Maybe send UriWrapper domain?